src/structures/Guild.js

```diff
@@ -1138,6 +1138,22 @@ class Guild extends Base {
     return this.name;
   }
 
+  toJSON() {
+    const json = super.toJSON([
+      'createdTimestamp',
+      'nameAcronym',
+    ], [
+      'presences',
+      'voiceStates',
+      'verificationLevel',
+      'explicitContentFilter',
+      'features',
+    ]);
```

Comment:
verification level, ecf, and features are all static information. For all of these methods, think about what the http api will give you, and ignore anything the ws gateway will give you.